app-emulation/gsplus/gsplus-0.14.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Modern cross-platform Apple IIgs emulator"
+DESCRIPTION="GSplus is an Apple IIgs emulator. It's an open-source project based on the KEGS and \
+GSPort emulators. It supports fullscreen, joysticks and audio through the SDL2 library across \
+platforms."
+HOMEPAGE="https://apple2.gs/plus/"
+COPYRIGHT="2003 Kent Dickey
+	2010-2014  GSport contributors
+	2016-2018  Dagen Brock"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="480572054518112647c8fae5d7ea7046a6d6ecfb"
+SOURCE_URI="https://github.com/digarok/gsplus/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="0b5d7cc83d66625f55f2c0518ec65f697a5a02af94fca018eb692d6a7eb87d2c"
+SOURCE_DIR="gsplus-$srcGitRev"
+PATCHES="gsplus-$portVersion.patchset"
+
+ARCHITECTURES="all"
```

Comment:
@OscarL is right of course. On the other hand, if this works with gcc2 (which I doubt) and is kept as such, all the usages of `$secondaryArchSuffix` should be cleaned up. It would always be empty without `SECONDARY_ARCHITECTURES`.

Reply:
Well, I had one day without st*pid mistakes ;) Will fix.